projects/canopy-test-app/src/app/app.module.ts

```diff
@@ -2,12 +2,16 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { ReactiveFormsModule } from '@angular/forms';
 import {
-  LgAccordionModule,
-  LgAlertModule,
+  LgAccordionComponent,
```

Comment:
I think this is slightly wrong, I would expect you to import only the `LgAccordionComponent` and that component will have the imports for the other ones. Same for all the other main components.

Reply:
That is a very good idea!

Reply:
Even if the idea by itself is very good, after doing some research on this, I came to the conclusion that it is impossible to export other components from a standalone component. So the only way to fix this would be to import every component individually. Please let me know in case you have any other ideas.